internal/services/report_processor.go

```diff
@@ -113,6 +114,14 @@ func ProcessReport(msg *kafka.Message) {
 				k8s_object_name,
 			)
 
+			if workload_metrics, err := model.GetWorkloadMetricsForTimestamp(experiment_name, interval_end); err != nil {
+				log.Errorf("Error while checking for workload_metrics record: %s", err)
+				continue
+			} else if !reflect.ValueOf(workload_metrics).IsZero() {
+				log.Debugf("worload_metrics table already has data for interval_end time: %v.", interval_end)
+				continue
```

Comment:
@patilsuraj767  - If metrics data at interval_end time is already available, do we still need to continue with further processing?

Reply:
Ohh, got it. If there are multiple slice data of 6hrs it should continue to process it.